config/software/lua.rb

```diff
@@ -15,16 +15,14 @@
 #
 
 name "lua"
-default_version "5.3.3"
-
-version("5.1.5") do
-  source md5: "2e115fe26e435e33b0d5c022e4490567"
-  license_file "COPYRIGHT" # dropped in 5.2.x
```

Comment:
So we need to make sure that we keep this `license_file` modification for 5.1.5, or just drop that version all together.

Reply:
sure